apps/web/app/root.tsx

```diff
@@ -1,86 +1,10 @@
-import {
-  Links,
-  Meta,
-  Outlet,
-  Scripts,
-  ScrollRestoration,
-  isRouteErrorResponse,
-} from 'react-router';
+import { Buffer } from 'safe-buffer';
 
-import { Flex, styled } from 'leather-styles/jsx';
+/**
+ * Polyfill global Buffer
+ * (should be called before another code)
+ */
+// @ts-expect-error // different types
+globalThis.Buffer = Buffer;
 
-import type { LeatherProvider } from '@leather.io/rpc';
-import leatherUiStyles from '@leather.io/ui/styles?url';
-
-import type { Route } from './+types/root';
-import stylesheet from './app.css?url';
-import { Footer } from './layouts/footer/footer';
-import { GlobalLoader } from './layouts/nav/global-loader';
-import { Nav } from './layouts/nav/nav';
-
-export function links() {
-  return [
-    { rel: 'stylesheet', href: stylesheet },
-    { rel: 'stylesheet', href: leatherUiStyles },
-  ] satisfies Route.LinkDescriptors;
-}
-
-declare global {
-  interface Window {
-    LeatherProvider?: LeatherProvider;
-  }
-}
-
-export function Layout({ children }: { children: React.ReactNode }) {
-  return (
-    <html lang="en">
-      <head>
-        <meta charSet="utf-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <Meta />
-        <Links />
-      </head>
-      <styled.body>
-        <GlobalLoader />
-        <Nav />
-        <Flex flexDir="column" marginLeft="navbar" minHeight="100vh">
-          <styled.main flex={1}>{children}</styled.main>
-          <Footer />
-        </Flex>
-        <ScrollRestoration />
-        <Scripts />
-      </styled.body>
-    </html>
-  );
-}
-
-export default function App() {
-  return <Outlet />;
-}
-
-export function ErrorBoundary({ error }: Route.ErrorBoundaryProps) {
-  let message = 'Oops!';
-  let details = 'An unexpected error occurred.';
-  let stack: string | undefined;
-
-  if (isRouteErrorResponse(error)) {
-    message = error.status === 404 ? '404' : 'Error';
-    details =
-      error.status === 404 ? 'The requested page could not be found.' : error.statusText || details;
-  } else if (import.meta.env.DEV && error && error instanceof Error) {
-    details = error.message;
-    stack = error.stack;
-  }
-
-  return (
-    <main>
-      <h1>{message}</h1>
-      <p>{details}</p>
-      {stack && (
-        <pre>
-          <code>{stack}</code>
-        </pre>
-      )}
-    </main>
-  );
-}
+export { links, Layout, default, ErrorBoundary } from './app';
```

Comment:
I think it's convention to leave the named link/meta/error boundary exports in the root tsx

So all good to move the layout components, but lets leave the rest here